Cargo.toml

```diff
@@ -21,6 +21,9 @@ travis-ci = { repository = "alteous/gltf" }
 gltf-json = "0.1"
 lazy_static = "0.2"
 
+[patch.crates-io]
```

Comment:
I didn't know that - I thought `patch` was the new feature to address this, but the docs specifically mention subcrates within one repo under `path`.